docker/docker-compose.yml

```diff
@@ -41,11 +41,9 @@ services:
       - api
     ports:
       - "3000:3000"
-    environment:
-      CHOKIDAR_USEPOLLING: true
     profiles:
       - webapp
 
 volumes:
   mssql_data:
-    driver: local
\ No newline at end of file
+    driver: local
```

Comment:
@rscarrasco could you please clarify this change? Is the docker build not working on your side without it? Or is something wrong with the database?

Reply:
it's [probably](https://docs.docker.com/storage/volumes/#start-a-service-with-volumes) `local` by default. so no need to specify it

Reply:
Ummm, I don't recall making that change. Its probably a mistake. I'll add it back.

Reply:
It seems I accidentally removed a newline at the end of the file. Sorry About, fixing now.

Reply:
Ah I see, it's about the new line only. I though you'd added the driver:local my bad.